OneBusAway/ui/search/MapSearchViewController.swift

```diff
@@ -185,6 +185,11 @@ class MapSearchViewController: OBAStaticTableViewController, UISearchResultsUpda
 
         let tabBarHeight: CGFloat = 49
         self.tableView.contentInset.bottom = tabBarHeight
-        self.tableView.scrollIndicatorInsets.bottom = tabBarHeight
+        if #available(iOS 11.1, *) {
```

Comment:
I'd rather see the deployment target get updated to 11.4 than an availability check for a minor version of iOS that came out in 2017.